source/queue_policy/policies.rst

```diff
@@ -176,10 +178,10 @@ Cloud Computing Allocations
 
 To request allocation for a new project, complete the `Cloud Computing
 Allocation form
-<https://docs.google.com/forms/d/e/1FAIpQLScbCVdipW-Bj2iD-bPzFjrzGjOdVM_jbmabbEZ3-CNrWdrdBA/viewform?usp=sf_link>`_`
+<https://docs.google.com/forms/d/e/1FAIpQLScbCVdipW-Bj2iD-bPzFjrzGjOdVM_jbmabbEZ3-CNrWdrdBA/viewform?usp=sf_link>`_.
 After you complete the form, create a Cloud
 help ticket to track the issue. Send email to
-rdhpcs.cloud.help@noaa.gov, copy to gonzalo.lassally@noaa.gov and
+rdhpcs.cloud.help@noaa.gov, copy to joshua.king@noaa.gov and
```

Comment:
We need to remove these names from the page. I'll make sure Josh can access the HD system. If we wanted, we can create a mail list for the additional people to email.  However, getting an email from the HD should be sufficient.